services/minespace-web/.env-example

```diff
@@ -14,3 +14,4 @@ BASE_PATH=
 ASSET_PATH=/
 
 VCAUTHN_PRES_REQ_CONF_ID=minespace-access-0.1-dev
+NODE_OPTIONS=--max_old_space_size=2048
```

Comment:
Just a note for this change, since devs manage their own copies of .env's, you'll want to make an announcement to the team to make sure they have these lines in their core/minespace .env's after the PR is merged.